bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import boto3
+
+
+class Read_Write_Files:
```

Comment:
class name should be like ReadWriteFiles. 
Ref: https://www.python.org/dev/peps/pep-0008/#class-names